synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -0,0 +1,186 @@
+import * as path from "path";
+import * as taskLib from "azure-pipelines-task-lib/task";
+import * as toolLib from "azure-pipelines-tool-lib";
```

Comment:
we can remove azure-pipelines-tool-lib as this library is not used anywhere in this class